js/reference.js

```diff
@@ -496,8 +509,7 @@ var ERMrest = (function(module) {
      * @returns {ERMrest.Reference} The copy of the reference object.
      */
     function _referenceCopy(source) {
-        // TODO: make a (copy-on-write) copy of the source reference
-        return source; // TODO
+        return Object.assign(Object.create(Reference.prototype), source);
```

Comment:
Since we are supporting IE, we should include the [Polyfill](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/assign#Polyfill) for `Object.assign`.